From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@simonrolfe__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@simonrolfe : Thanks for your contribution to the Azure documentation! The author, @adamglick, has been notified to review your proposed change.

adamglick commented:
#sign-off

adamglick commented:
Thanks @simonrolfe!
@tysonn This is approved; please accept and merge.

tysonn commented:
@adamglick - can you submit a pull request to simon's fork to remove the dot and slash from the article links in the modified text? When you link from one article to another article in the same GitHub folder, you need only the file name - the dot and slash are not needed. The [wiki contains info on how to  make a change to another users fork](https://microsoft.sharepoint.com/teams/azurecontentguidance/wiki/Pages/Make%20changes%20to%20another%20person%27s%20pending%20pull%20request.aspx).

simonrolfe commented:
Added a second commit which removes the unnecessary ./ prefixes from same-folder links in the article. Hopefully that's useful!

simonrolfe commented:
I've updated the PR title to more accurately reflect both of the commits - do I need to do anything else?

adamglick commented:
#sign-off

adamglick commented:
@simonrolfe Good stuff.  accepted.
@tysonn This is good to go, he's fixed the ./ removal.  Ready to merge.

Commit:
Update TOC for MVC, move rules engine ref

Commit:
Edit for MVC and grammar

Commit:
Merge branch 'release-mvc-cdn' of https://github.com/MicrosoftDocs/azure-docs-pr into release-mvc-cdn

Commit:
task 899: MVC updates

Commit:
task 899: fix merge error

Commit:
Merge branch 'release-mvc-cdn' of https://github.com/MicrosoftDocs/azure-docs-pr into release-mvc-cdn

Commit:
task 899: update Enable CDN for MVC

Commit:
task 899: Edits and redo screen shot to include browser

Commit:
Merge branch 'release-mvc-cdn' of https://github.com/MicrosoftDocs/azure-docs-pr into release-mvc-cdn

Commit:
task 902: Edit tutorial, update TOC

Commit:
task 902: Fix link to free trial include

Commit:
task 898: Remove note in overview per MVC guidance

Commit:
task 898: Overview edits

Commit:
Merge branch 'release-mvc-cdn' of https://github.com/MicrosoftDocs/azure-docs-pr into release-mvc-cdn

Commit:
task 899: incorporate MVC review comments

Commit:
task 898: Incorporate Overview MVC review comments

Commit:
task 902: Incorporate custom domain tutorial MVC review comments

Commit:
task 902: Incorporate custom domain tutorial MVC review comments

Commit:
task 902: Add profile/endpoint prereq

Commit:
task 902: Fix merge error

Commit:
Merge branch 'release-mvc-cdn' of https://github.com/MicrosoftDocs/azure-docs-pr into release-mvc-cdn

Commit:
task 899: Incorporate quickstart CFR comments

Commit:
Merge branch 'release-mvc-cdn' of https://github.com/MicrosoftDocs/azure-docs-pr into release-mvc-cdn

Commit:
task 899: Update screenshots to meet standards

Commit:
task 905: Fix merge issue in TOC

Commit:
task 899: Update Clean up resources section

Commit:
task 899: Fix procedure format, add rectangle to screenshot

Commit:
Merge branch 'release-mvc-cdn' of https://github.com/MicrosoftDocs/azure-docs-pr into release-mvc-cdn

Commit:
Merge branch 'release-mvc-cdn' of https://github.com/MicrosoftDocs/azure-docs-pr into release-mvc-cdn

Commit:
Task 902: Incorporate CFR review comments and move cdnverify step to new topic

Commit:
Merge branch 'release-mvc-cdn' of https://github.com/MicrosoftDocs/azure-docs-pr into release-mvc-cdn

Commit:
Task 902: Reorganize articles and redo screenshots

Commit:
Merge branch 'release-mvc-cdn' of https://github.com/MicrosoftDocs/azure-docs-pr into release-mvc-cdn

Commit:
Merge branch 'release-mvc-cdn' of https://github.com/MicrosoftDocs/azure-docs-pr into release-mvc-cdn

Commit:
task 902: incorporate review comments, add GoDaddy screenshots

Commit:
task 902: edits

Commit:
Merge branch 'release-mvc-cdn' of https://github.com/MicrosoftDocs/azure-docs-pr into release-mvc-cdn

Commit:
fix merge issues

Commit:
fix merge issues

Commit:
Merge branch 'release-mvc-cdn' of https://github.com/MicrosoftDocs/azure-docs-pr into release-mvc-cdn

Commit:
Task 898: Incorporate MVC review comments

Commit:
updates for the rebranding of OMS

ShawnJackson committed:
edit pass: ms-confluence-jira-plugin

ShawnJackson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into ms-confluence-jira-plugin

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into v-craic0326

Commit:
added performance tips for Rx Java SDK

Added additional tips specific to Rx Java SDK (from the github page)
Removed references to Direct Connectivity and links that do not exists in docs.microsoft right now.
Next iteration : Add required links

Commit:
changes "rx java" to "async java"

Commit:
rectified links

rectified links in performance tips for java

Commit:
added link

added link to Rx Java performance tips

Commit:
minor edits

ShawnJackson committed:
edit pass: ms-confluence-jira-plugin

ShawnJackson committed:
edit pass: ms-confluence-jira-plugin

Commit:
Merge branch 'release-mvc-cdn' of https://github.com/MicrosoftDocs/azure-docs-pr into release-mvc-cdn

Commit:
Minor link (anchor) correction

Commit:
Fix issues in first-round MVC articles

Commit:
Fix procedures and remove article from TOC

Commit:
Cleanup and edits

Commit:
updates after review

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into v-craic0326

ktoliver committed:
edit pass: three SAP HANA articles

ktoliver committed:
more edits

dlepow committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into fix

Commit:
updates

Commit:
changed links

- changed order of links to performance tips
- changed reference to documentdb headers documentation to cosmosdb

Commit:
Remove TOC.md and update toc.yml

Commit:
changed order of links

Commit:
changed order of links

Commit:
added reference to performance-tips-rx-java.md

Commit:
Fix blocking issues

ktoliver committed:
Merge pull request #32529 from dksimpson/release-mvc-cdn

Initial update of CDN docs for MVC

Commit:
resolved conflicts and added performance-tips-rx-java.md to TOCs

v-alje committed:
Bringing even with master.

ktoliver committed:
more edits

ktoliver committed:
more edits

v-alje committed:
Bringing even with master.

Commit:
changed rx-jva to async-java

- changed name of file to performance-tips-async-java.md and changed related references
- changed setPageSize to setMaxItemCount
- changed recommended connection pool size from "200-1000" to "2000-3000" for higher throughput

cherylmc committed:
update

v-alje committed:
Bringing even with master.

v-nagta committed:
USER STORY 1225140 SaaS App Tutorial: Teamwork update

v-nagta committed:
update

v-alje committed:
Bringing even with master.

HeidiSteen committed:
document limits

v-alje committed:
Bringing even with master.

buffalo-sauce-rules committed:
Update toc.yml

ninarn committed:
updated elastic pool article and images to match current service and experience

ninarn committed:
Merge remote-tracking branch 'upstream/master'

v-nagta committed:
update

rwike77 committed:
Adding link to cloud services migration

KumudD committed:
removed preview text

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-limits

HeidiSteen committed:
Storage limits in the include, object maximums are in the  main article

HeidiSteen committed:
footnote revisions

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-limits

HeidiSteen committed:
H2 edits

HeidiSteen committed:
HTML formatting

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-limits

HeidiSteen committed:
more formatting

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-limits

ninarn committed:
fixing merge conflicts

ninarn committed:
fixing link issue

Commit:
automation: dsc config issue update

v-shils committed:
Merge pull request #36986 from KumudD/includefileupdate0403

removed preview text

cynthn committed:
Update capture-image.md

Commit:
Update index.md

cherylmc committed:
update

v-shils committed:
Merge pull request #36984 from rwike77/tocupdate

Adding link to cloud services migration

cherylmc committed:
update

billmath committed:
adding art for remaining new files

cherylmc committed:
update

PRMerger13 committed:
Merge pull request #37021 from cherylmc/cli

update

v-shils committed:
Merge pull request #36742 from georgewallace/autosolutionupdates

automation : updates to inventory doc

v-shils committed:
Merge pull request #37011 from georgewallace/dscconfig

automation: dsc config issue update

Commit:
Task 1045: Add unique cache note to standard tier

v-alje committed:
Bringing even with master.

CarlRabeler committed:
Update sql-database-long-term-backup-retention-configure.md

Commit:
made changes to maxPoolSize tip.

Commit:
minor changes in formatting

Commit:
required updates after PR review

ktoliver committed:
Merge pull request #36974 from ninarn/master

updating the sql elastic pool article

Commit:
Merge pull request #37028 from CarlRabeler/patch-47

Update sql-database-long-term-backup-retention-configure.md

Commit:
minor change to code snippet

Commit:
minor change to code snippet

markgalioto committed:
Updating the Backup TOC with changes to SQL db section

Commit:
Fixed Manage NIC entry.

ShawnJackson committed:
Merge pull request #36672 from ktoliver/1217545-a

edit pass: three SAP HANA articles

dlepow committed:
N-series: Win drivers

dlepow committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into fix

cherylmc committed:
graphics

ktoliver committed:
Merge pull request #36297 from craigcaseyMSFT/v-craic0326

updates for the rebranding of OMS

Ja-Dunn committed:
Merge pull request #36434 from ShawnJackson/ms-confluence-jira-plugin

edit pass: ms-confluence-jira-plugin

Commit:
Correct CDN product names

ktoliver committed:
Merge pull request #36965 from jinzh-azureiot/patch-3

Update toc.yml

sanjeev3 committed:
Update toc.yml

sanjeev3 committed:
Update samples-dotnet.md

ktoliver committed:
Merge pull request #37030 from jimdial/patch-126

Fixed Manage NIC entry.

dlepow committed:
Batch: link and command fixes

dlepow committed:
edit

sanjeev3 committed:
Update samples-dotnet.md

sanjeev3 committed:
Update toc.yml

PRMerger16 committed:
Merge pull request #37032 from dlepow/gpu

N-series: Win drivers

v-shils committed:
Merge pull request #36939 from cherylmc/s2s

update

ktoliver committed:
Merge pull request #37029 from markgalioto/apr42018

Updating the Backup TOC with changes to SQL db section

billmath committed:
updating

Commit:
Fix formatting

Commit:
updating gopher image

ktoliver committed:
Merge pull request #36394 from srinathnarayanan/patch-7

added performance tips for Rx Java SDK

Commit:
Merge pull request #36469 from scottcate/patch-2

Minor link (anchor) correction

jpconnock committed:
Updating manager metadata from Tim to JP

sanjeev3 committed:
Update overview.md

dlepow committed:
Acrolinx fixes

sanjeev3 committed:
Update overview.md

sanjeev3 committed:
Update overview.md

ktoliver committed:
Merge pull request #37033 from dksimpson/dean-1045

 Task 1045: Add unique cache note to standard tier

tynevi committed:
Merge pull request #37037 from MicrosoftDocs/adkinn-patch-3

updating gopher image

georgewallace committed:
corrected code formatting

ktoliver committed:
Merge pull request #36894 from v-nagta/teamwork

USER STORY 1225140 SaaS App Tutorial: Teamwork update

tynevi committed:
Merge pull request #37043 from tysonn/master

Gopher

Commit:
Fix blocking issues

PRMerger15 committed:
Merge pull request #37039 from jpconnock/managerUpdate

Updating manager metadata from Tim to JP

charwen committed:
added "Telmex Uninet"

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-limits

CarlRabeler committed:
Update sql-database-automatic-tuning-enable.md

charwen committed:
add Telmex Uninet

PRMerger17 committed:
Merge pull request #37042 from georgewallace/patch-2

automation: corrected code formatting

jpconnock committed:
Manager Update from Tim to JP, pt 2

jpconnock committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into managerUpdate

PRMerger18 committed:
Merge pull request #37044 from charwen/patch-54

added "Telmex Uninet"

PRMerger18 committed:
Merge pull request #37046 from charwen/patch-55

add Telmex Uninet

PRMerger19 committed:
Merge pull request #37045 from CarlRabeler/patch-48

Update sql-database-automatic-tuning-enable.md

HeidiSteen committed:
metadata

sanjeev3 committed:
Update samples-dotnet.md

sanjeev3 committed:
Update overview.md

Ja-Dunn committed:
Merge pull request #37034 from dlepow/fix

Batch: link and command fixes

Commit:
Merge pull request #37014 from cynthn/patch-74

Update capture-image.md

Ja-Dunn committed:
Merge pull request #37048 from dksimpson/release-mvc-cdn

Fix blocking issues

Commit:
Merge pull request #37052 from jpconnock/managerUpdate

Manager metadata update

GitHubber17 committed:
Merge pull request #37025 from billmath/pimApril

adding art for remaining new files

jpconnock committed:
Updating Manager metadata, prt 3

jpconnock committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into updateManager

GitHubber17 committed:
Merge pull request #36877 from MicrosoftDocs/release-mvc-cdn

[Azure CDN] Initial MVC rework release

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

JasonWHowell committed:
Fixing broken forum link.

Ja-Dunn committed:
Merge pull request #37040 from sanjeev3/master

Updated overview, TOC, and sample pages

Ja-Dunn committed:
Merge pull request #37054 from jpconnock/updateManager

Update manager

GitHubber17 committed:
Merge pull request #36955 from HeidiSteen/heidist-limits

document limits

JasonWHowell committed:
Updating broken forum links throughout Stream Analytics.

v-alje committed:
Merge pull request #37056 from MicrosoftDocs/FromPublicRepo

From public repo

PRMerger20 committed:
Merge pull request #37057 from JasonWHowell/streamanalytics1

Fixing broken forum links in Stream Analytics

Commit:
Merge pull request #37059 from JasonWHowell/streamanalytics2

Updating broken forum links throughout Stream Analytics.
